### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by praveek**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/release-notes/index.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ Note that all these extensions must be updated together. Flutter and React plugi
 * Analytics shared state is made available on boot-up if there are any previously stored identifiers.
 * Shared state is only updated when an identifier is set and not on get requests.
 
+### Android Campaign Standard 2.0.2
+
+* Made extension compatible with Kotlin 1.4.
```
Comment (calebk1) [inline src/pages/documentation/release-notes/index.md:47]: "Made the extension compatible..."

File: src/pages/documentation/adobe-journey-optimizer/release-notes.md [hunk 2]
```diff
@@ -6,10 +6,10 @@
 
 * Adds support for Adobe Journey Optimizer powered in-app messages.
 
-### Android AEPMessaging 2.0.1
+### Android Messaging 2.0.1
 
 * Adds support for Adobe Journey Optimizer powered in-app messages.
-* Adds support for AEP 2.x extensions. Note, AEPMessaging 2.0.1 is not backwards compatible with the ACP 1.x extensions.
+* Adds support for AEP 2.x extensions. Note, Messaging 2.0.1 is not backwards compatible with the ACP 1.x extensions.
```
Comment (calebk1) [inline src/pages/documentation/adobe-journey-optimizer/release-notes.md:12]: "Please note that Messaging 2.0.1 is not..."

"... with the ACP-prefixed 1.x extensions."

(Also, is it 2.0.1? It's not 2.0 that starts that?)

**Review by calebk1 — APPROVED** [review]
Looks good! Just a couple of small comments.

File: src/pages/documentation/release-notes/index.md [hunk 3]
```diff
@@ -50,26 +54,115 @@ Note that all these extensions must be updated together. Flutter and React plugi
...
+* Major version update for [Identity for Edge Network](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-edgeidentity-android).
+
+### Android UserProfile 2.0.0
+
+- Major version update for [User Profile](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-userprofile-android).
+- The following APIs have been deprecated and will be removed in a future release:
+
+  | Deprecated API                      | Recommended Alternative              |
+  | ----------------------------------- | ------------------------------------ |
+  | `UserProfile.registerExtension()`   | `MobileCore.registerExtensions()`    |
```
Comment (swarna04) [inline src/pages/documentation/release-notes/index.md:152]: Nit: This applies to all the extensions and it's covered in migration guide. Do we still need i there?

File: src/pages/documentation/adobe-journey-optimizer/release-notes.md (hunk 2 above)
Comment (emdobrin) [reply src/pages/documentation/adobe-journey-optimizer/release-notes.md:12]: ... is not backwards compatible with MobileCore 1.x and compatible extensions. 
(same for the general notes)

Comment (swarna04) [inline src/pages/documentation/adobe-journey-optimizer/release-notes.md:12]: I'm a little confused here, shouldn't this be for 2.0.0?

**Comment by swarna04** [comment]
I see updates to mostly main release notes here, assuming the respective extension ones are updated already. Just want to confirm.

File: src/pages/documentation/adobe-journey-optimizer/release-notes.md (hunk 2 above)
Comment (praveek) [reply src/pages/documentation/adobe-journey-optimizer/release-notes.md:12]: I am going to remove this line.

File: src/pages/documentation/release-notes/index.md (hunk 3 above)
Comment (praveek) [reply src/pages/documentation/release-notes/index.md]: I have removed this from release notes, as we have covered it in other places.

File: src/pages/documentation/adobe-journey-optimizer/release-notes.md (hunk 2 above)
Comment (praveek) [reply src/pages/documentation/adobe-journey-optimizer/release-notes.md]: I have removed this line, as we don't mention it for other extensions. We have also covered this in other places.

File: src/pages/documentation/release-notes/index.md [hunk 4]
```diff
@@ -38,10 +38,14 @@ Note that all these extensions must be updated together. Flutter and React plugi
 
 ### Android Analytics 2.0.1
 
-* Made extension compatible with Kotlin 1.4.
+* Made the extension compatible with Kotlin 1.4.
```
Comment (emdobrin) [inline src/pages/documentation/release-notes/index.md:41]: nit: can you make this update to individual rn too?

**Review by emdobrin — APPROVED** [review]

**Pull request merged**